src/Sylius/Bundle/InstallerBundle/Command/SetupCommand.php

```diff
@@ -26,14 +26,9 @@
 class SetupCommand extends AbstractInstallCommand
 {
     /**
-     * @var array
+     * @var Currency
```

Comment:
It does not have an interface.
It extends `Symfony\Component\Validator\Constraint` which is an abstract class.